ui/src/pages/OBProxy/ClusterList.tsx

```diff
@@ -0,0 +1,90 @@
+import type { ObproxyOBProxyOverview } from '@/api/generated';
+import { Link } from '@umijs/max';
+import { Button, Card, Col, Table, Typography } from 'antd';
+import type { ColumnsType } from 'antd/es/table';
+
+interface ClusterListProps {
+  handleAddCluster: () => void;
+  obproxies: ObproxyOBProxyOverview[] | undefined;
+  loading: boolean;
+}
+
+const { Text } = Typography;
+
+const columns: ColumnsType<ObproxyOBProxyOverview> = [
+  {
+    title: 'OBProxy 资源名',
+    dataIndex: 'name',
+    key: 'name',
+    render: (value, record) => (
+      <Text ellipsis={{ tooltip: value }}>
+        <Link to={`${record.namespace}/${record.name}`}>{value}</Link>
+      </Text>
+    ),
+  },
+  {
+    title: 'OBProxy 集群名称',
+    dataIndex: 'proxyClusterName',
+    key: 'proxyClusterName',
+  },
+  {
+    title: '对应 OB 集群',
+    dataIndex: 'obCluster',
+    key: 'obCluster',
+    render: (value) => <Text>{value.name}</Text>,
+  },
+  {
+    title: '版本',
+    dataIndex: 'image',
+    key: 'image',
+  },
+  {
+    title: 'Replicas',
+    dataIndex: 'Replicas',
+    key: 'Replicas',
+  },
+  {
+    title: 'serviceIp',
+    dataIndex: 'serviceIp',
+    key: 'serviceIp',
+  },
+  {
+    title: 'creationTime',
```

Comment:
Use Chinese here. The same as `replicas`, `serviceIP` and `status` fields.

Reply:
Done.